docs/user-guide/services.md

```diff
@@ -355,22 +356,30 @@ they desire, but leaves them freedom to do discovery in their own way.
 Applications can still use a self-registration pattern and adapters for other
 discovery systems could easily be built upon this API.
 
-## External services
-
-For some parts of your application (e.g. frontends) you may want to expose a
-Service onto an external (outside of your cluster, maybe public internet) IP
-address.  Kubernetes supports two ways of doing this: `NodePort`s and
-`LoadBalancer`s.
+## Service Types
 
 Every `Service` has a `type` field which defines how the `Service` can be
-accessed.  Valid values for this field are:
+accessed. Default and base type is `ClusterIP`, which exposes a service
+on connection from inside the cluster.
+
+Valid values for `ServiceType` field are:
 
    * `ClusterIP`: use a cluster-internal IP only - this is the default and is
-     discussed above
-   * `NodePort`: use a cluster IP, but also expose the service on a port on each
-     node of the cluster (the same port on each node)
-   * `LoadBalancer`: use a ClusterIP and a NodePort, but also ask the cloud
-     provider for a load balancer which forwards to the `Service`
+     discussed above. Choosing this value means that you want this to be the only
```

Comment:
Your expansions here are good (thanks).  I think the flow from "Service Types" to "External Services" to "Details about Service Types" is a bit awkward.

What if this restructured as:

```
## Publishing services - service types

For some parts of your app...

Every service has a type...

### Type NodePort

```

It still feels a little redundant to describe the types in fair detail only to follow up with a sub-chapter on each type half a page later...